airflow/www/views.py

```diff
@@ -808,11 +808,11 @@ def dag_details(self, session=None):
             tags=tags
         )
 
-    @expose('/rendered')
+    @expose('/rendered_templates')
```

Comment:
1. If we're going to change the URL of this view, I'd suggest we move to hyphens instead of underscores while we're at it. Hyphens are more of a standard pattern these days.
2. Did you make this plural on purpose? The existing template navigation/heading is singular.
3. Do we have to worry about aliasing/redirecting the old route to the new?

```suggestion
    @expose('/rendered-templates')
```